zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+* Substantive changes to this ZIP defining the zBloc.
+* Changes to the set of zBloc Constituencies and/or the distribution of Voting Units among those Constituencies.
+* Changes to the policy a Constituency uses for its decision-making.
+
+The ZIP Editors will modify ZIP 0 to define other categories of change to Zcash Governance that MUST or SHOULD require Approval of a Governance Proposal.
+
+Unrelated changes SHOULD NOT be bundled into a single Governance Proposal. This does not prevent a single Governance Proposal from being used to decide whether to adopt a proposed Network Upgrade involving multiple consensus and/or process changes, since those are considered related. During the process of drafting Governance Proposals, the ZIP Editors MAY request that proposals be split in order to satisfy this rule, or merged if they are sufficiently interdependent.
+
+### Disputes
+
+If any zBloc Constituency, or the ZIP Editors, have concerns relating to the conduct of any vote, they MAY raise a dispute. Possible reasons to raise a dispute include believing that:
+
+* A Constituency or any holders of keys for its Voting Units were unduly pressured or coerced.
+* A Constituency voted contrary to its intended position (due to key compromise or error, for instance), or contrary to its stated decision-making policy.
+* A Constituency failed to disclose a relevant conflict of interest.
+* There were technical irregularities in the use of the voting mechanism.
+
+If a dispute is raised, the Constituencies SHALL first resolve any technical issues as far as possible (e.g. by rotating keys), and then make one or more Governance Proposals intended to resolve the situation to the satisfaction of the community.
+
+### Exceptional decisions to remediate security vulnerabilities
+
+The governance processes specified in this ZIP are not intended to prevent consensus node implementors from making consensus changes that they reasonably believe are critical to maintain the security of the Zcash Mainnet network and/or Zcash users. Such changes might need to be developed and released according to a controlled disclosure process that precludes public voting.
+
+In cases where the zBloc governance processes could not be used for this reason, the consensus node implementors MUST, once the risk of exploitation has passed, publish a thorough post-mortem analysis that explains why this was the case.
+
+## Funding Proposals
+
+### Zcash Community Fund
+
+A funding stream will be established for a Zcash Community Fund, consisting of 20% of the block subsidy paid to a 2-of-3 P2SH multisig with keys held by the Financial Privacy Foundation, the Zcash Foundation and the Electric Coin Company ("Key-holder Organisations").
+
+This funding stream will start immediately on activation of this ZIP, which is assumed to be after the expiry of the existing ``FS_FPF_ZCG`` and ``FS_DEFERRED`` funding streams. It is proposed to be continued until modified by a future network upgrade.
```

Comment:
Raising again that I don't like the funding mechanism to be coupled to the governance mechanism. Also, is the FPF intended to be included here?

What is the rationale behind the keyholders being different from the constituencies? I can see where the coinholder vote would need its disbursement power to be split among the other constituencies, but otherwise it's not obvious why we should have separate keyholders. Add a rationale if this is actually the chosen design.